OverviewPhoto/client/src/components/ImageCarousel.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Proptypes from 'prop-types';
+import { displayPhotoModal, displayCarouselModal } from '../action/actions';
+import style from '../overview.module.css';
+
+class ImageCarousel extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentId: null,
+    };
+  }
+
+  componentDidMount() {
+    const { currentStoreId } = this.props;
+    document.getElementById(`carouselModalImage${currentStoreId.toString()}`).classList.add('d-flex');
+    document.getElementById(`carouselModalImage${currentStoreId.toString()}`).classList.remove('d-none');
+    this.setState({ currentId: currentStoreId });
+  }
+
+  onNext() {
+    const { photos } = this.props;
+    const { currentId } = this.state;
+    const newId = currentId >= photos.length ? 1 : currentId + 1;
+    document.getElementById(`carouselModalImage${newId.toString()}`).classList.add('d-flex');
+    document.getElementById(`carouselModalImage${newId.toString()}`).classList.remove('d-none');
+    document.getElementById(`carouselModalImage${currentId.toString()}`).classList.add('d-none');
+    document.getElementById(`carouselModalImage${currentId.toString()}`).classList.remove('d-flex');
+    this.setState({ currentId: newId });
```

Comment:
Implementation note: setState is used here instead of Redux because of Redux latency when player spams button clicks.